mmdet/models/bbox_heads/bbox_head.py

```diff
@@ -187,13 +188,45 @@ def refine_bboxes(self, rois, labels, bbox_preds, pos_is_gts, img_metas):
 
         Returns:
             list[Tensor]: Refined bboxes of each image in a mini-batch.
+
+        Example:
+            >>> # xdoctest: +REQUIRES(module:kwarray)
```

Comment:
As a temporary solution, `kwarrary` can be added as a test-time dependency in `tests/requirements.txt`.
For future usage except for unittests, I suggest porting it to mmcv.